src/main/java/com/baidu/hugegraph/structure/Element.java

```diff
@@ -34,6 +34,10 @@ public int hashCode() {
 
     @Override
     public boolean equals(Object other) {
+        if (this == other) {
+            return true;
+        }
+
```

Comment:
judge class equals

Reply:
Already judged with "
       if (!(other instanceof Element)) {
            return false;
        }"

Reply:
return false when two instances of class Element with same id and different class